tests/conftest.py

```diff
@@ -29,8 +29,8 @@ def assert_render_template(template, context, expected, request=None):
     return assert_render_template
 
 
-@pytest.fixture
-def assert_parse_error():
+@pytest.fixture(params=["rusty", "django"])
+def assert_parse_error(request):
```

Comment:
Why is this called `request`? It seems very misleading to me - I think of a Django `request`.

Reply:
Agree, but I have no control over that. This is the documented way to access the fixture `params`

https://docs.pytest.org/en/stable/how-to/fixtures.html#parametrizing-fixtures